tests/dryrun_test.go

```diff
@@ -149,34 +148,30 @@ var _ = Describe("[sig-compute]Dry-Run requests", decorators.SigCompute, func()
 	})
 
 	Context("VirtualMachines", func() {
-		var vm *v1.VirtualMachine
-		resource := "virtualmachines"
-
-		newVM := func() *v1.VirtualMachine {
-			vmiImage := cd.ContainerDiskFor(cd.ContainerDiskCirros)
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(vmiImage, "echo Hi\n")
-			vm := libvmi.NewVirtualMachine(vmi)
-			return vm
-		}
+		var (
+			vm        *v1.VirtualMachine
```

Comment:
Why do we need this variable?

Reply:
We're using it all over

Reply:
I believe we can just inline this

Reply:
Not sure I follow. This follows the ginkgo pattern of define a var, assign it on `BeforeEach` and use it on tests. Why is it different then all other cases? Do I miss something?

Reply:
I find it little bit error prone and redundant especially once the `BeforeEach` is one liner. Upon looking again I am fine with the namespace but still find the `vm` redundant.